intg-androidtv/apps.py

```diff
@@ -14,6 +15,12 @@
     "Ziggo": {"url": "ziggogo://"},
     "Videoland": {"url": "videoland-v2://"},
     "Steam Link": {"url": "steamlink://"},
+    "Waipu TV": {"url": "waipu://tv"},
+#    "Magenta TV": {"url": "https://www.telekom.de/magenta-tv"},
+    "Zatoo": {"url": "zatoo://zattoo.com"},
```

Comment:
Typo in name and url (app cannot be launched). Correct:
- Zattoo
- zattoo://zattoo.com